Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py

```diff
@@ -57,6 +67,7 @@
 FETCH_SIZE = int(param.get('fetch_size', 50))
 INSECURE = not param.get('insecure', False)
 TIME_METHOD = param.get('time_method', 'Simple-Date')
+TIME_FORMAT = TIME_FORMATS_DICT.get(param.get('time_format', '%Y-%m-%d %H:%M:%S'))
```

Comment:
Isn't this "more accurate"?
```suggestion
TIME_FORMAT = TIME_FORMATS_DICT.get(param.get('time_format'), '%Y-%m-%d %H:%M:%S')
```